webapp/frontend/src/pages/_document.tsx

```diff
@@ -1,13 +1,24 @@
-import { Html, Head, Main, NextScript } from 'next/document'
+import { Head, Html, Main, NextScript } from "next/document";
```

Comment:
is this just formatting changes or does have some impact to the code as well?

Reply:
Probably my editor configurations caused that, can we have something like a prettier config files so that autoformatting are all consistent across teams?

Reply:
sure. please look into that and add to your PR as well. We should already have prettier in the code base so probably missing some other configurations